workers/main/src/common/Redmine.ts

```diff
@@ -0,0 +1,75 @@
+import * as mysql from 'mysql2/promise';
+import { Pool, PoolOptions, RowDataPacket } from 'mysql2/promise';
+
+import { ProjectUnit } from './types';
+
+export class Redmine {
+  private pool: Pool;
```

Comment:
Let's separate class than manages pool and class that handles requests sql requests